### ISSUE TITLE
[Bug] pfSense widget TypeError: Cannot read properties of undefined (reading 'status')

### ISSUE THREAD

**Issue by cpuks** [gethomepage/homepage#2165]
### Description

```
Something went wrong.
TypeError: Cannot read properties of undefined (reading 'status')

    at u (http://192.168.88.21:3002/_next/static/chunks/3483.a8f45082b57991ed.js:1:1154)
    at h (http://192.168.88.21:3002/_next/static/chunks/6133-1abb682bd7f256a6.js:1:35344)
    at n (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:15374)
    at L (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:28855)
    at div
    at li
    at q (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:31545)
    at ul
    at J (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:35301)
    at div
    at http://192.168.88.21:3002/_next/static/chunks/6133-1abb682bd7f256a6.js:1:44035
    at div
    at l (http://192.168.88.21:3002/_next/static/chunks/6133-1abb682bd7f256a6.js:1:70166)
    at http://192.168.88.21:3002/_next/static/chunks/6133-1abb682bd7f256a6.js:1:62780
    at http://192.168.88.21:3002/_next/static/chunks/6133-1abb682bd7f256a6.js:1:61725
    at l (http://192.168.88.21:3002/_next/static/chunks/6133-1abb682bd7f256a6.js:1:70166)
    at http://192.168.88.21:3002/_next/static/chunks/6133-1abb682bd7f256a6.js:1:41981
    at div
    at W (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:35696)
    at div
    at div
    at he (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:53112)
    at n (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:15374)
    at http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:90174
    at ue (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:51118)
    at div
    at div
    at div
    at be (http://192.168.88.21:3002/_next/static/chunks/pages/index-62d3754e96f28eb6.js:1:60469)
    at u (http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:14817)
    at s (http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:14609)
    at a (http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:16243)
    at a (http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:15886)
    at s (http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:16802)
    at u (http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:15314)
    at http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:90174
    at http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:13875
    at l (http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:1471)
    at n (http://192.168.88.21:3002/_next/static/chunks/pages/_app-9363fc24f9e34742.js:1:6048)
    at r (http://192.168.88.21:3002/_next/static/chunks/main-5004eaa7b9f962cb.js:1:6989)
    at Y (http://192.168.88.21:3002/_next/static/chunks/main-5004eaa7b9f962cb.js:1:9236)
    at ie (http://192.168.88.21:3002/_next/static/chunks/main-5004eaa7b9f962cb.js:1:11878)
```

### Steps to reproduce

    - pfSense:
        icon: pfsense.png
        href: https://pfsense.domain:5555
        widget:
            type: pfsense
            url: https://pfsense.domain:5555
            username: homepage
            password: password
            wan: bge0

### homepage version

0.7.3

### Installation method

Docker

### Troubleshooting

I've tried different versions of pfsense-api - from 1.6.0 to 1.6.4 (latest) - same error.
Also I checked if I can curl with user credentials that I use for homepage and it works
![image](https://github.com/gethomepage/homepage/assets/21194723/440117d9-6ba2-4de6-96b0-d7d2a3e87966)
![image](https://github.com/gethomepage/homepage/assets/21194723/dadc1e59-4b37-4212-97a1-15ab3f1803ad)
![image](https://github.com/gethomepage/homepage/assets/21194723/357c2f4e-d5ea-4602-bbbf-d357ea697262)

### Before submitting, I have made sure to

- [X] Check [the documentation](https://gethomepage.dev/)
- [X] Follow [the troubleshooting guide](https://gethomepage.dev/latest/more/troubleshooting/) (please include output above if applicable).
- [X] Search [existing issues](https://github.com/gethomepage/homepage/search?q=&type=issues) and [discussions](https://github.com/gethomepage/homepage/search?q=&type=discussions).

**Comment by shamoon** [comment]
Again, nothing has changed. We don’t need screenshots of the pfsense api, you need to try reaching it from the homepage container, and using the same credentials.

Again, doubt it’s broken, willing to bet it’s networking issue

**Comment by cpuks** [comment]
Does it only work for plain HTTP and not for HTTPS?

**Comment by shamoon** [comment]
I don’t know why it would. Generally, homepage doesn’t ‘care’ and will accept self-signed certs so as long as a url returns the expected data things should work. That said, I don’t have an actual pfsense widget with which to test this

**Comment by cpuks** [comment]
Thanks - I've send request to authenticate and found out that API responds with different hwif name than pfsense uses - pppoe0 instead of bge0 in my case. Would be nice to add there notice about getting output from postman / curl / thuderclient or simillar and look out for hwif of WAN section
![image](https://github.com/gethomepage/homepage/assets/21194723/e4b6f263-b1c1-458d-ab8e-d357399b19fa)
Thanks again for pointing me into right direction with unifi as well (shim interface) - and sorry for wasting your time.

**Comment by shamoon** [comment]
Unless Im misunderstanding your comment about curl-ing the data, that is what the troubleshooting guide suggests? https://gethomepage.dev/v0.7.3/more/troubleshooting/#service-widget-errors

**Comment by cpuks** [comment]
Well yes and no - people including myself usually take shortcuts and in this case I just used interface that I've used wee while ago - bge0, then I assumed WAN interface is bge0 instead of pppoe0 in my case as I have FTTH. After I finally curled data and looked at it - I finally figured out what's the problem. Maybe all options like bge/ige/pppoe etc should be listed there or this should be idiot proof guide to skip questions as mine. Thanks
